op-challenger/runner/metrics.go

```diff
@@ -96,6 +102,7 @@ func NewMetrics(runConfigs []RunConfig) *Metrics {
 		metrics.consecutiveFailuresCurrent.WithLabelValues(runConfig.Name).Set(0)
 		metrics.panicsTotal.WithLabelValues(runConfig.Name).Add(0)
 		metrics.invalidTotal.WithLabelValues(runConfig.Name).Add(0)
+		metrics.timeoutsTotal.WithLabelValues(runConfig.Name).Add(0)
```

Comment:
We'll need to update the dashboard and alerting to pick this up.

Probably we should combine `invalidTotal`, `panicsTotal` and `timeoutsTotal` into a single `vmFailure` metric with a `reason` label that we set to either `incorrectStatus`, `timeout` or `panic`.  That way we can update the alerts once to flag if any of these increase rather than having to have separate alerts for each.

Reply:
Done. The metrics are now `vmFailuresTotal{reason="..."}`, `setupFailuresTotal`, and `consecutiveSetupFailures`.